lib/api/controllers/authController.js

```diff
@@ -79,21 +79,22 @@ class AuthController {
     const strategy = request.input.args.strategy;
 
     return this.kuzzle.passport.authenticate({body: request.input.body, query: request.input.args, original: request}, strategy)
+      .then(userObject => this.kuzzle.pluginsManager.trigger('auth:strategyAuthenticated', {strategy, content:userObject}))
```

Comment:
we emit a JSON object with the strategy the user object, to enable the pipe listener to handle different stuff according to the actual strategy if needed.